MNE/disp3D/engine/model/data3Dtreemodel.cpp

```diff
@@ -492,7 +491,7 @@ SensorDataTreeItem* Data3DTreeModel::addSensorData(const QString& sSubject,
                                         const MNEBemSurface& tBemSurface,
                                         const FiffInfo& fiffInfo,
                                         const QString& sDataType,
-                                        const double dCancelDist,
+                                        const double& dCancelDist,
```

Comment:
making a reference to a double is not really beneficial. since the address itself is as large as an int + it has to be resolved internally - so I would recommend not using a reference to a double
https://stackoverflow.com/questions/270408/is-it-better-in-c-to-pass-by-value-or-pass-by-constant-reference

"except for builtin types (char, int, double, etc.),"